CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/RunPane.java

```diff
@@ -87,7 +87,7 @@ public final class RunPane extends BorderPane implements KeyListener {
     private static final String FILTER_STYLE_ALERT = "-fx-background-color: red; -fx-text-fill: black;-fx-prompt-text-fill:grey;";
 
     private final ImportController importController;
-    private final TableView<TableRow> sampleDataView = new TableView<>();
+    protected final TableView<TableRow> sampleDataView = new TableView<>();
```

Comment:
instead of making this protected directly, are you able to keep this private and instead make a protected getter for it?